src/main/java/org/killbill/billing/plugin/adyen/client/payment/service/AdyenPaymentServiceProviderHostedPaymentPagePort.java

```diff
@@ -100,9 +100,14 @@ public Map getDirectory(final String merchantAccount,
         return directoryClient.getDirectory(params);
     }
 
-    // Used to verify completion
-    @SuppressWarnings("deprecation")
```

Comment:
Why this change? `Signer#signFormParameters` (`SHA1` version) is deprecated.

Reply:
Added back.